PopulateDatabase/data.json

```diff
@@ -38,16 +38,125 @@
     ],
     "bikes": [
       {
-        "id": "5bedc8ebdd97c96c09a41b08",
-        "available": "true",
-        "manufacturer": "Van Raam",
-        "model": "VX4L",
-        "hourlyCost": 10,
+        "manufacturer": "Schwinn",
+        "model": "Schwinn Women's Cruiser",
+        "hourlyCost": 1,
         "type": "tandem",
         "address": "1 Microsoft Way, Redmond, WA",
         "ownerUserId": "16a46619738a4865a5afcb5e18ffb138",
         "suitableHeightInMeters": 0.75,
-        "maximumWeightInKg": 5.5
+        "maximumWeightInKg": 5.5,
+        "imageUrl": "https://contosobikerental.blob.core.windows.net/public/images/sample-bike-2.jpg"
```

Comment:
You can now update the imageUrls listed here to include more variety.  Also note that the imageUrls should end with 2 digits.  E.g. "-1.jpg" -> "-01.jpg"

Reply:
Great, I'll do in a subsequent update

Reply:
>> imageUrls should end with 2 digits
Got it. I'll update those references now.

Reply:
I updated to include to point to new bike URLs as well as ensure 2 digits.